Packs/VMwareWorkspaceONEUEM/Integrations/VMwareWorkspaceONEUEM/VMwareWorkspaceONEUEM.py

```diff
@@ -0,0 +1,644 @@
+    if last_seen:
+        last_seen = dateparser.parse(last_seen).strftime(READABLE_DATE_FORMAT)  # type: ignore
+
+    hr_dict = {
+        CONSTANT_STRING["DEVICE_FRIENDLY"]: response.get('friendlyName', ''),
+        "UUID": response.get('uuid', ''),
+        "Platform": response.get('platformInfo', {}).get('platformName', ''),
+        "Model": response.get('platformInfo', {}).get('modelName'),
+        "Ownership": enrollment_info.get('ownership', ''),
+        CONSTANT_STRING["SERIAL_NUM"]: response.get('serialNumber', ''),
+        CONSTANT_STRING["MAC_ADDR"]: response.get('macAddress', ''),
+        CONSTANT_STRING["COMPLIANCE_STATUS"]: compliance,
+        CONSTANT_STRING["USER_EMAIL"]: enrollment_info.get('userEmailAddress', ''),
+        CONSTANT_STRING["LAST_SEEN"]: last_seen
+    }
+    hr = tableToMarkdown("Device", hr_dict,
+                         [CONSTANT_STRING["DEVICE_FRIENDLY"], "UUID", "Platform", "Model", "Ownership",
+                          CONSTANT_STRING["SERIAL_NUM"],
+                          CONSTANT_STRING["MAC_ADDR"], CONSTANT_STRING["COMPLIANCE_STATUS"],
+                          CONSTANT_STRING["USER_EMAIL"], CONSTANT_STRING["LAST_SEEN"]],
+                         removeNull=True)
+    return remove_empty_elements_for_context(camel_to_pascal(response)), hr
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    client.http_request(method='GET', url_suffix=URL_SUFFIX['TEST'])
+    return 'ok'
+
+
+def vmwuem_devices_search_command(client: Client, args: dict) -> CommandResults:
+    """
+    Searches devices using the search API according to the arguments
+
+    :type client: ``Client``
+    :param client: client to use
+    :type args: ``dict``
+    :param args: arguments from demisto.args
+
+    :return: Command results containing the outputs and context.
+    :rtype: ``CommandResults``
+    """
+    # Prepare parameters for request
+    args = remove_empty_elements(args)
+    params = validate_and_parameterize_devices_search_arguments(args)
+
+    # Make the call.
+    response = client.http_request(method='GET', url_suffix=URL_SUFFIX['SEARCH_DEVICES'], params=params)
+
+    if not response.text:
+        return CommandResults(readable_output=MESSAGES['NO_RECORDS_FOUND'].format('device'))
+
+    # Prepare context and human readable
+    json_response = response.json()
+    outputs, readable_output = prepare_context_and_hr_for_devices_search(json_response)
+    return CommandResults(outputs_prefix=OUTPUT_PREFIX['SEARCH_DEVICES'], outputs_key_field="Uuid", outputs=outputs,
```

Comment:
Please arrange this command that it will be easier to read (as the commands below).
Also Let's remove the global parameters here: `OUTPUT_PREFIX['SEARCH_DEVICES']` and use the original instead in all command, for example:
```
outputs_prefix='VMwareWorkspaceONEUEM.Device'
```

It is easier to read and understand.

Reply:
@Shellyber This is something we're following as a best practice to create a common OUTPUT_PREFIX. It benefits us to make the changes to the endpoint URL if used in multiple methods. Currently, we have only 3 commands but considering that we can have more commands in the future.